CRM/Contribute/BAO/Contribution.php

```diff
@@ -4340,6 +4340,16 @@ public static function completeOrder($input, $ids, $contribution, $isPostPayment
     $contribution->contribution_status_id = $contributionParams['contribution_status_id'];
 
     CRM_Core_Error::debug_log_message('Contribution record updated successfully');
+    $contributionSoft = civicrm_api3('ContributionSoft', 'get', [
+      'sequential' => 1,
+      'contribution_id' => $contribution->id,
+    ]);
+
+    $contributionSoft = (object) $contributionSoft['values'][0];
+    //Send notification to owner for PCP
+    if ($contributionSoft->pcp_id) {
+      CRM_Contribute_Form_Contribution_Confirm::pcpNotifyOwner($contribution, $contributionSoft);
```

Comment:
This function is only called from one place in the code so we can easily change it to receive an array rather than cast to an object